src/metatensor/models/utils/model_io.py

```diff
@@ -0,0 +1,65 @@
+import importlib
+from typing import Dict, List
+
+import torch
+
+
+def save_model(
+    arch_name: str,
+    model: torch.nn.Module,
+    hypers: Dict,
+    all_species: List[int],
+    path: str,
+) -> None:
+    """Saves a model to a file, along with all the metadata needed to load it.
+
+    Parameters
+    ----------
+        arch_name (str): The name of the architecture.
+
+        model (torch.nn.Module): The model to save.
+
+        hypers (Dict): The hyperparameters used to train the model.
+
+        all_species (List[int]): The list of all species that the model can handle.
```

Comment:
The indentation looks weird. Also we do not need the the types here. Maybe adjust your ChatGPT prompt that you most likely used for this.

Reply:
Copilot

Reply:
This is numpy's docstring style, which we are not using in any other repo. I would go with the sphinx standard:

```suggestion
    :param arch_name: The name of the architecture.
    :param model: The model to save.
    :param hypers: The hyperparameters used to train the model.
    :param all_species: The list of all species that the model can handle.
```

and then the return value

```
    :return: <some description here>
```

If sphinx is setup right, the types should be taken directly from the type annotation, so no need to repeat them here.